COLLADA2GLTF/GLTF/GLTFMesh.cpp

```diff
@@ -83,7 +83,7 @@ namespace GLTF
             size_t attributesCount = this->getMeshAttributesCountForSemantic(semantic);
             for (size_t j = 0 ; j < attributesCount ; j++) {
                 shared_ptr <GLTF::GLTFAccessor> selectedMeshAttribute = this->getMeshAttribute(semantic, j);
-                unsigned int indexSet = j;
+                unsigned int indexSet = (int)j;
```

Comment:
Yeah, switching it over to int (or unsigned int) should be fine. Passing an int to a function that takes size_t shouldn't throw a warning since size_t is bigger.